scripts/deployment/deployment_utils.sh

```diff
@@ -52,6 +53,8 @@ function deploy_canisters() {
     dfx_deploy ${airdrop_canister_name} --network=${network} --argument="'(${env_vars})'"  
   elif [ "$canister_only" = "rs" ]; then
     dfx_deploy ${rs_canister_name} --network=${network} --argument="'(${env_vars})'"
+  elif [ "$canister_only" = "decideid" ]; then
+    dfx_deploy ${decideid_canister_name} --network=${network} --argument="'(${env_vars})'"
```

Comment:
I believe we must have different arguments for DecideID project, as example, just because it has no connection with RS, Vesting, etc.